cli/Cargo.toml

```diff
@@ -14,9 +14,14 @@ edition = "2018"
 [dependencies]
 ascii_table = "3.0"
 base64 = "0.13"
+chrono = "0.4"
 clap = "2.33.1"
+elasticsearch = "7.11.0-alpha.1"
```

Comment:
is there a particular reason we are using the alpha version and not a stable one?

Reply:
It is the latest tag on their releases https://github.com/elastic/elasticsearch-rs/releases. Theres no stable one yet